src/components/radios/_macro.spec.js

```diff
@@ -122,6 +122,16 @@ const EXAMPLE_RADIO_ITEM_RADIOS = {
     },
 };
 
+const EXAMPLE_RADIO_ITEM_WITH_PRESET_TEXTAREA = {
+    other: {
+        otherType: 'textarea',
+        checked: true,
+        id: 'other-textbox-radio-with-text-input',
+        name: 'other-answer',
+        value: 'other-answer',
```

Comment:
```suggestion
        value: 'other answer',
```
This should be an answer someone would write in the text area so for it to be a good example it shouldn't be hyphenated